pages/common/pveperf.md

```diff
@@ -0,0 +1,8 @@
+# pveperf
+
+> Proxmox benchmark.
```

Comment:
Could you please elaborate on this a bit more? As someone who doesn't know Proxmox I have no idea what this command does.

Reply:
```suggestion
> A benchmarking tool in proxmox server. Gather CPU/hard disk performance data on the hard disk mounted at PATH (/ is used as default).
```